docs/js-delivery-test-object.md

```diff
@@ -0,0 +1,114 @@
+---
+id: js-delivery-test-object
+title: Test object parameter reference in Mojito JS Delivery
+sidebar_label: Test object parameters
+---
+
+The test object contains everything an experiment needs to run, including the trigger, tracking information and recipe code (or variants). When you pass all this information into the ```Mojito.addTest()``` function, Mojito will execute all the logic in your experiment based on its order of execution.
+
+## Example test objects in JS & YAML
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--JavaScript-->
+```js
+Mojito.addTest({
+    "id": "w12",
+    "name": "Example test",
+    "divertTo": "1",
+    "sampleRate": 1,
+    "state": "live",
+    "gaExperimentId": "dsdy2h872g7d32h782n2",
+    "trigger": function (testObject) {
+        if (document.location.pathname === "/") {
+            // Activate the test at DOM Content Loaded
+            Mojito.utils.domReady(testObject.activate);
+        }
+    },
+    "recipes": {
+        "0": {
+            "name": "control"
+        },
+        "1": {
+            "name": "treatment",
+            "js": function(testObject) {
+                alert("You're in an experiment...");
+            },
+            "css": "body{display:none;}"
+        }
+    }
+});
+```
+
+<!--YAML-->
+```yml
+id: w12
+name: Example test
+divertTo: "1"
+sampleRate: 1
+state: live
+gaExperimentId: dsdy2h872g7d32h782n2
+trigger: trigger.js
+recipes:
+    "0":
+        name: control
+    "1":
+        name: treatment
+        js: 1.js
+        css: 1.css
+```
+
+<!--YAML trigger.js-->
+```js
+function trigger(testObject) {
+    if (document.location.pathname === "/") {
+        // Activate the test at DOM Content Loaded
+        Mojito.utils.domReady(testObject.activate);
+    }
+}
+```
+
+<!--YAML 1.js-->
+```js
+function(testObject) {
+    alert("You're in an experiment...");
+}
+```
+
+<!--YAML 1.css-->
+```css
+body {
+    display:none;
+}
+```
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+
+## Test object: Root level
+
+Parameter | Description
+--|--
+**css** <br> Type: *string* <br> *Optional* | Shared CSS for the test object that is applied to all recipes. <br>`JS format`: Expects a JS string of CSS styles. <br>`YAML format`: Expects a relative path to a CSS file.
+**divertTo** <br> Type: *string* <br> *Optional* | Enable divert mode to send all eligible traffic into a particular recipe's key.
+**id** <br> Type: *string* <br> *Required* | A canonical test object ID by which subjects' assignments are recorded against. 
+**js** <br> Type: *function* <br> *Required* | Shared JS for the test object that is applied to all recipes. <br>`JS format`: Expects a valid JavaScript function on this key. <br>`YAML format`: Expects a relative path to a JS file with that function.
+**name** <br> Type: *string* <br> *Required* | The name of the test object that's useful in tracking.
+**recipes** <br> Type: *object* <br> *Required* | An object containing the definition of available recipes.
```

Comment:
Add a link on "object" to the Recipe Object section
Likewise add a link on "definition" to the Receipe objects sub-level section